frontend/assets/stylesheets/components/_elevated-banner.scss

```diff
@@ -0,0 +1,96 @@
+/* ==========================================================================
+   Elevated Banner
+   ========================================================================== */
+.elevated-banner {
+  @supports (clip-path : polygon(0% 100%, 0% 0%, 100% 0%, 100% 95.73%)) {
```

Comment:
Although Firefox supports the `clip-path` property, unfortunately it doesn't currently support the `polygon` value except behind a dev flag. The diagonal stuff will only work on Safari and Chrome for the moment =(.